mlflow/pyfunc/loaders/chat_model.py

```diff
@@ -74,7 +84,12 @@ def predict(
             Model predictions in :py:class:`~ChatCompletionResponse` format.
         """
         messages, params = self._convert_input(model_input)
-        response = self.chat_model.predict(self.context, messages, params)
+        parameters = inspect.signature(self.chat_model.predict).parameters
+        if "context" in parameters or len(parameters) == 3:
+            _logger.info(_DROP_CONTEXT_IN_CHAT_MODEL_PREDICT_INFO.format(func_name="predict"))
```

Comment:
If this line is called repeatedly, would that cause a flood of logs?

Reply:
Emm yes... do we need to turn this into warnings.warn 🤔

Reply:
you can use a global flag.